src/coreclr/jit/typelist.h

```diff
@@ -3,7 +3,11 @@
 
 #define GCS EA_GCREF
 #define BRS EA_BYREF
+#if defined(TARGET_WASM)
+#define PS TARGET_POINTER_SIZE
```

Comment:
This can be `TARGET_POINTER_SIZE` unconditionally

Reply:
ok, push to runtime?

Reply:
Yes, that may be nice.